build.gradle

```diff
@@ -1260,6 +1266,8 @@ if (file(cachedBundledRuntime).exists()) {
     defaultStubRuntime = cachedStandaloneRuntime
 } else if (BUILD_CLOSED) {
     defaultStubRuntime = cachedBundledRuntime
+} else if (IS_STUB_RUNTIME_OPENJFX) {
```

Comment:
this won't be used if the cachedBundleRuntime exists. Shouldn't the order be reversed? (first check IS_STUB_TUNTIME_OPENJFX)

Reply:
Either is fine with me: if you want to make this change it could be the first test instead of the last.